CosNet.WebUI/Private/Cosplay/CosplayList.razor

```diff
@@ -41,13 +58,13 @@
 @code {
     private IEnumerable<CosplayDTO> cosplays;
 
-    protected override async Task OnInitializedAsync()
-    {
-        cosplays = await CosplayService.GetCosplaysAsync();
-    }
+                        protected override async Task OnInitializedAsync()
+                        {
+                            cosplays = await CosplayService.GetCosplaysAsync();
+                        }
 
-    public void ClickedToId(Guid cosplayId)
-    {
-        NavigationManager.NavigateTo($"cosplay/{cosplayId.ToString()}");
-    }
-}
+                        public void ClickedToId(Guid cosplayId)
+                        {
+                            NavigationManager.NavigateTo($"cosplay/{cosplayId.ToString()}");
+                        }
+                    }
```

Comment:
Why so much indentation?

Reply:
well that was because of copy paste sometimes it does this weird shit